Example/Podfile

```diff
@@ -7,7 +7,7 @@ target 'Emission' do
                :subspecs => %w( Core RCTText RCTWebSocket RCTNetwork ),
                :inhibit_warnings => true
 
-  if ENV['ARTSY_STAFF_MEMBER'] != nil || ENV['CI'] != nil
+  if true #ENV['ARTSY_STAFF_MEMBER'] != nil || ENV['CI'] != nil
```

Comment:
;)

got opinions on exposing the host's ENV vars to the CP mac app? I guess we could do it then ensure CP's ENVs are set up after